rca/static_src/sass/components/_introduction.scss

```diff
@@ -24,6 +24,14 @@
 
         &--no-margin {
             margin-bottom: 0;
+
+            .enquire-form & {
```

Comment:
Can we add a modifier class instead? It'd be nice to keep within the BEM guidelines if possible.

Reply:
Updated!